changelogs/fragments/56900-route53-facts-check-mode.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - route53_facts - add support for check mode
```

This formulation still sounds like this is a new feature. I think it's better to phrase it as follows:
```suggestion
  - route53_facts - the module did not advertise check mode support, causing it not to be run in check mode.
```